drafts/2016-01-12.md

```diff
@@ -0,0 +1,42 @@
+# IPFS Weekly #2
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open.  In these posts, we will try to highlight some of the development that happened in the past week.  For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [github](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+## Updates
+
+* [**(specs)**](//github.com/ipfs/specs) There was much debate in the [IPLD spec](https://github.com/ipfs/specs/pull/37) about pathing notation. [@mildred](//github.com/mildred) in particular did a lot of good work.
+* [**(go-ipfs)**](//github.com/ipfs/go-ipfs) The IPFS Docker image is now tested automatically on Travis, thanks to [@ChristianKniep](//github.com/ChristianKniep) and [@whyrusleeping](//github.com/whyrusleeping). For more, see the [go-ipfs#2064](//github.com/ipfs/go-ipfs/pull/2064) pull request.
+* [**(js-ipfs)**](//github.com/ipfs/js-ipfs) Now you can do `npm i -g js-ipfs` and use jsipfs bootstrap + id + version commands, fully compatible with the go-ipfs repo, thanks to [@diasdavid](//github.com/diasdavid)!
+* [**(community)**](//github.com/ipfs/community) [@RichardLitt](//github.com/RichardLitt) merged the PR for [JavaScript guidelines](//github.com/ipfs/community/blob/master/js-contribution-guidelines.md) for IPFS repos.
+
+## Work in Progress
+
+* [**(distributions)**](//github.com/distributions) [@Dignifiedquire](//github.com/Dignifiedquire) further developed the distributions page.
```

Comment:
unless @dignifiedquire objects, can we include a link to 

http://v04x.ipfs.io/ipfs/QmZyvWokPYGg6DrjE6o2V7qhThzZQZ8QCWqdd2U3S75HXC/index.html

it's so pretty!